webpack.config.base.js

```diff
@@ -4,35 +4,10 @@
 
 import path from 'path';
 import webpack from 'webpack';
-import fs from 'fs';
 import { dependencies as externals } from './app/package.json';
-import { dependencies as possibleExternals } from './package.json';
-
-// Find all the dependencies without a `main` property and add them as webpack externals
-function filterDepWithoutEntryPoints(dep: string): boolean {
```

Comment:
Removing this function makes the entire app treated as webpack externals?

Reply:
Not sure, I replaced the file with what was working in 'erb-video-example', but I just re-tested this without these changes and there was no problem. Check out a latest commit reverting these changes.